gallery/src/styles/PageLayout/PageLayoutPage.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+
+import { GalleryDescriptionTile } from '../../gallery-components/GalleryTiles';
+import RawHtmlExample from '../../CodeExample';
+
+const SingleColumnLayoutExampleCode = require('!!raw-loader!./SingleColumnLayoutExample.html').default,
+    TwoColumnLayoutExampleCode = require('!!raw-loader!./TwoColumnLayoutExample.html').default,
+    ReactRootLayoutExampleCode = require('!!raw-loader!./ReactRootLayoutExample.html').default;
+
+const PageLayoutGuidelinesPage = () =>
+  <>
+    <GalleryDescriptionTile>
+      <p>
+        HTML and SCSS are provided for both single and two column page layouts. We use{' '}
+        <a href="https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox">
+          flexbox
+        </a>
+        to handle our layouts.
+      </p>
+      <p>
```

Comment:
Don't all these need `.nx-p`?

Reply:
> Don't all these need `.nx-p`?

Yes. The `<p>` tags are the one place where I want to go back on my "never style HTML tags rule". Not sure why `<p>` tags bug me in particular. https://github.com/sonatype/sonatype-react-shared-components/pull/28/commits/e982547a0fd7c62941a7a6975f5b823f25ee551b

Reply:
> Yes. The <p> tags are the one place where I want to go back on my "never style HTML tags" rule.

You missed the one this comment is on.

> Not sure why <p> tags bug me in particular.

Probably because there's a lot of them.

> e982547

Wrong commit btw

Reply:
> Probably because there's a lot of them.

One approach that consuming apps could take, including the gallery even, would be to use scss like `p { @extend .nx-p; }` if they are sure they want nx styles globally.  We just can't have it that way out of the box.

Or as another idea, we chould create some "root" class (`nx` maybe) that consuming apps should add to the top of each part of their app that uses RSC styles, and then we could add styles for `.nx p` and so on.

Reply:
> > Probably because there's a lot of them.
> 
> One approach that consuming apps could take, including the gallery even, would be to use scss like `p { @extend .nx-p; }` if they are sure they want nx styles globally. We just can't have it that way out of the box.
> 
> Or as another idea, we chould create some "root" class (`nx` maybe) that consuming apps should add to the top of each part of their app that uses RSC styles, and then we could add styles for `.nx p` and so on.

https://github.com/sonatype/sonatype-react-shared-components/pull/28/commits/4186f9636ce50591b5adf13d305112df1a2c240e